### ISSUE TITLE
[Feature]: Is it possible to build termux-app with pacman as the default package manager?

### ISSUE THREAD

**Issue by shaunmulligan** [termux/termux-app#3248]
### Feature description

I have successfully built bootstraps.zip using the pacman option, but now having installed that in my build of termux-app, I get the following error when trying to install apps:
```
/data/data/com.termux/files/usr/bin/pkg: line 348: pacman: command not found
```

Or do I just need to include pacman in the bootstraps.zip build?

### Additional information

N/A

**Comment by Maxython** [comment]
What command did you use to build bootstrap based on pacman?

**Comment by shaunmulligan** [comment]
I used your PR https://github.com/termux/termux-packages/pull/14692 to generate the bootstrap

**Comment by shaunmulligan** [comment]
@Maxython any tips on getting pacman to work? Your PR with `--pm pacman` is the only way I have managed to build a workable bootstrap.zip. Everything else fails or creates a zip that is too large to load. I now just need to figure out how to actually get `pkg install` to work :P

**Comment by Maxython** [comment]
So I understand you are using `build-bootstraps.sh` to create bootstrap, but the thing is that this bootstrap generator does not work correctly. I recommend that you use `generate-bootstraps.sh` (or use ready-made - https://github.com/termux-pacman/termux-packages/releases).

As far as building bootstrap based on pacman, you need to use the `--pm` flag, because for bootstrap to work properly (on some basis) need to make settings for the package managers, which is what this flag does. There are no other ways to change the package manager in bootstrap.

**Comment by Maxython** [comment]
If you want to add additional packages to bootstrap, then you can use the `--add` flag.

**Comment by shaunmulligan** [comment]
Yeah I'm using that already but it only works for packages in the official repo, but I need to add in a package and scripts specific to my application. At the moment I'm just hacking the generate script to look in the local `output` folder for .deb if the package doesn't exist on the repository URL. Its a mess but it looks like its working 👍

**Comment by agnostic-apollo** [comment]
Set log level to verbose and check logcat after reinstalling app to see if pacman was even installed from the bootstrap.

https://github.com/termux/termux-app#Debugging
